apps/x509.c

```diff
@@ -605,6 +608,10 @@ int x509_main(int argc, char **argv)
         case OPT_CHECKIP:
             checkip = opt_arg();
             break;
+        case OPT_BUNDLE:
+            bundle = 1;
+            noout = 1;
```

Comment:
Also here, better not add `noout = 1`.

Reply:
Removing this will cause any output to apply to the last certificate found, which is probably not what would be intended.  I'm "ok" with removing it, but just wanted to note the caveat.